ChangeLog

```diff
@@ -191,6 +191,15 @@ Release date: TBA
   (Ie. not necessarily at the end)
 
 
+What's New in Pylint 2.12.3?
+============================
+Release date:
+
+* Fixed handling of missing 'ancestors' object attribute.
+
+  Closes #5646
+
+
```

Comment:
We don't have a bug fix branch for 2.12 currently so we do not back-port those fix. This is something that we'd need to put into place after we release 2.13 but right now this is going to be in 2.13.

```suggestion

* Fixed handling of missing 'ancestors' object attribute.

  Closes #5646
```